libkbfs/md_journal.go

```diff
@@ -362,12 +362,18 @@ func (j mdJournal) putExtraMetadata(
 		return errors.New("Invalid extra metadata")
 	}
 
+	// TODO: We lose extraV3.wkbNew and extraV3.rkbNew here. Store
```

Comment:
Ah, this slipped past the first batch (the one I have a PR out for) but not the subsequent ones. I added more checks, though, so that the first batch would have caught it. But I'm fixing it in this PR, and merging after CI.